frontend/src/components/worksheets/Worksheet.js

```diff
@@ -687,6 +707,29 @@ class Worksheet extends React.Component {
         });
     }
 
+    delete() {
+        if (!window.confirm("Are you sure you want to delete this worksheet?")) {
```

Comment:
Add elaboration: "(Note that this does not delete the bundles, but just detaches them from the worksheet.)"